api/api_full.go

```diff
@@ -164,6 +164,13 @@ type FullNode interface {
 	// If oldmsgskip is set, messages from before the requested roots are also not included.
 	ChainExport(ctx context.Context, nroots abi.ChainEpoch, oldmsgskip bool, tsk types.TipSetKey) (<-chan []byte, error) //perm:read
 
+	// ChainCheckBlockstore performs an (asynchronous) health check on the chain/state blockstore
+	// if supported by the underlying implementation.
+	ChainCheckBlockstore(context.Context) error //perm:read
```

Comment:
Since this writes to a file in splitstore dir, and isquite expensive, I'd make the perm 'admin' ('write' could also work, but it's more of a node maintenance call, so admin fits better)

Reply:
ah yes, very good point; fixing.

Reply:
done!